addons/account/models/account_invoice.py

```diff
@@ -1693,6 +1635,35 @@ def _prepare_refund(self, invoice, date_invoice=None, date=None, description=Non
             values['date'] = date
         if description:
             values['name'] = description
+
+        # Treat refund tax lines. They should match the incoice's, except for the data
```

Comment:
what's an incoice?
review comment: what we do is copy data from invoice move from refund except the account_id and the tag_ids

Reply:
It's a small bean found on some island on the south of Asia. It's very tasty, and used in a lot of traditional meals. Never heard of incoice noodles?